### PR TITLE
feat: enable and configure environment_indicator module

### PR DESCRIPTION
**Pull request by lazysoundsystem**
Refs: OPS-8939

An attempt to match https://github.com/UN-OCHA/response-site/pull/459 - for Common Design, as a prelude to rolling it out for all sites. 

I'm not sure whether the associated ansible commit - https://github.com/UN-OCHA/tools-ansible/commit/9fd3c64e4b8f4d9a917db734dc2809d88c411cba - is necessary. For Response, the ansible directives appear to turn off the configured color for prod and I'm not sure if that's what we want or not.

### CODE REVIEW

**Comment by lazysoundsystem** [comment]
@rupl - a low-priority bump on this - just checking it's going as intended before rolling out to the other sites

**Comment by rupl** [comment]
I'm almost certain the problem is between the chair and keyboard, but I am unable to build this branch locally with various exotic composer errors such as 

```
  [Symfony\Component\Process\Exception\ProcessTimedOutException]
  The process "mv '/var/www/html/themes/custom' '/home/docker/.composer/cache/preserve-paths/2bd00c1fc4fa44d1ccd58915584bd4570ee1827a/ce53e059d054830a9573024dbf39
  dcc76f4d0816'" exceeded the timeout of 300 seconds.
```

I tried deleting that folder and all sorts of gymnastics, but anyway I have pushed a feature branch with your work, will let ECR build it for me, and deploy to our feature env to test it out next week.

File: config/environment_indicator.switcher.feature.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+langcode: en
+status: true
+dependencies: {  }
+machine: feature
+description: null
+name: Feature
+weight: '6'
+url: 'https://feature.commondesign-unocha-org.ahconu.org'
+fg_color: '#db7b18'
+bg_color: '#ffffff'
```
Comment (rupl) [inline config/environment_indicator.switcher.feature.yml:11]:
```suggestion
fg_color: '#ffffff'
bg_color: '#db7b18'
```

I believe the foreground/background got switched here. I've manually applied in my browser and took the screenshot below. Does it still provide enough distinction for you to see the difference, @lazysoundsystem ?

env-indicator-colors

**Review by rupl — APPROVED** [review]
Looks good, with the one change prepared inline (pending your approval).

File: config/environment_indicator.switcher.feature.yml (hunk 1 above)
Comment (lazysoundsystem) [reply config/environment_indicator.switcher.feature.yml]: Thanks @rupl - good catch, and the difference between the colors is nice and clear to me. I've updated the PR

**Pull request merged**